README.md

```diff
@@ -8,15 +8,19 @@
 
 ## 👵 Welcome Home!
 
-Runhouse gives your code the superpower of traversing remote infrastructure, so you
-can iterate and debug your ML apps and workflows locally in regular Python (no DSLs, yaml, or prescriptive
-dev environment) with full-scale compute and data (no sandbox). It's the fastest way to build, run,
-and deploy production-quality ML apps and workflows on your own infrastructure, and perhaps the only way to
-take production code and run it as-is locally (again, running on identical powerful infra) to iterate it further or
-debug.
-
-After you've sent a function or class to remote compute, Runhouse also allows you to persist, reuse, and share it as
-a service, turning otherwise redundant AI activities into common modular components across your team or company.
+Runhouse enables rapid, cost-effective Machine Learning development across research and production. 
+It allows you to dispatch Python functions and classes to your cloud compute infrastructure, and call them 
```

Comment:
maybe "to your cloud compute infrastructure of choice", or "desired cloud compute infrastructure"? think its worth driving home how agnostic we are